lib/microservices_engine.rb

```diff
@@ -40,26 +41,36 @@ def build=(b)
       @build = b
     end
 
+    # Returns the engine's current build
     def build
-      @build ||= '1.1.1'
+      @build ||= '0.0.0'
     end
 
+    # Returns the engine's YML configuration, alias for `reload_config`
```

Comment:
I originally had the intention of having three methods each with slightly different functionality, but decided against it right before I left for the day. I'll look into the aliasing and just use that instead.